static/src/javascripts/projects/commercial/modules/comscore.ts

```diff
@@ -39,35 +34,41 @@ const initOnConsent = () => {
  * Initialise comscore, industry-wide audience tracking
  * https://www.comscore.com/About
  */
-const setupComscore = () => {
+const setupComscore = (): Promise<void> => {
 	if (commercialFeatures.comscore) {
```

Comment:
Maybe an early return here would help with the deep indentation

Reply:
Good call